extension/src/plots/index.ts

```diff
@@ -48,7 +49,7 @@ export class Plots extends BaseRepository<TPlotsData> {
     this.experiments = experiments
 
     this.plots = this.dispose.track(
-      new PlotsModel(this.dvcRoot, experiments, workspaceState)
```

Comment:
[F] Removing experiments from here so that we can't get into a race condition where the selected revisions change mid-update.